### PR TITLE
Minor corrections to documentation and str for custom types

### PR DESCRIPTION
**Pull request by shilohbeeler-NI**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).

~~- [ ] I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.~~

~~- [ ] I've added tests applicable for this pull request~~

### What does this Pull Request accomplish?

Formatted string output for WaveformInfo and MeasurementStats now have more consistent formatting and minor inconsistencies in the fetch_measurement_stats documentation were rectified.

### What testing has been done?

Built successfully on local machine.

### CODE REVIEW

**Comment by sbethur** [comment]
ok to test

File: generated/niscope/niscope/measurement_stats.py [hunk 1]
```diff
@@ -17,7 +17,9 @@ def __repr__(self):
             'stdev={}'.format(self.stdev),
             'min_val={}'.format(self.min_val),
             'max_val={}'.format(self.max_val),
-            'num_in_stats={}'.format(self.num_in_stats)
+            'num_in_stats={}'.format(self.num_in_stats),
+            'channel={}'.format(self.channel),
+            'record={}'.format(self.record)
```
Comment (sbethur) [inline generated/niscope/niscope/measurement_stats.py:22]: Consulted with @thejcannon - `__repr__` should not include objects that are not part of class constructor.

Comment (shilohbeeler-NI) [reply generated/niscope/niscope/measurement_stats.py:22]: Got it. Reverting those changes.

File: generated/niscope/niscope/measurement_stats.py [hunk 2]
```diff
@@ -33,11 +35,11 @@ def __str__(self):
         if self.record is not None:
             string_representation += row_format_d.format('record', self.record)
 
-        string_representation += row_format_g.format('result', self.result)
```
Comment (sbethur) [inline generated/niscope/niscope/measurement_stats.py:36]: Need to fix the casing for `channel` and `record` too.

File: src/niscope/custom_types/waveform_info.py [hunk 3]
```diff
@@ -92,17 +94,17 @@ def __str__(self):
...
         string_representation += row_format_g.format('Absolute X0', self.absolute_initial_x)
         string_representation += row_format_g.format('Relative X0', self.relative_initial_x)
         string_representation += row_format_g.format('dt', self.x_increment)
-        string_representation += row_format_g.format('offset', self.offset)
-        string_representation += row_format_g.format('gain', self.gain)
+        string_representation += row_format_g.format('Offset', self.offset)
+        string_representation += row_format_g.format('Gain', self.gain)
         if self.samples is not None:  # We explicitly look for not None to differentiate from empty array
-            string_representation += row_format_g.format('wfm length', len(self.samples))
+            string_representation += row_format_g.format('Waveform Length', len(self.samples))
```
Comment (sbethur) [inline src/niscope/custom_types/waveform_info.py:105]: These changes would technically be considered as breaking. But I think it's ok to update them. @marcoskirsch, thoughts?

Comment (marcoskirsch) [reply src/niscope/custom_types/waveform_info.py:105]: I think it's fine. The same could be said for error codes returned by the runtime changing. I don't expect clients to parse the returned string.

**Pull request merged**